InvenTree/part/models.py

```diff
@@ -1628,31 +1627,6 @@ def clear_bom(self):
         """
         self.bom_items.all().delete()
 
-    def getRequiredParts(self, recursive=False, parts=None):
```

Comment:
Question: Why is this removed? I use it in multiple plugins, is there something wrong with it?